src/storage/persistence.rs

```diff
@@ -57,7 +56,14 @@ pub fn use_persistent<T: Serialize + DeserializeOwned + Default + Clone + 'stati
             state
         }
     };
-    cx.use_hook(|| UseStorageEntry::new(state))
+    let state_clone = state.clone();
+    let state_signal = state.data;
+    use_selector(cx, move || {
```

Comment:
You can use the use_effect hook here instead of use_selector because you don't care about the return value